docs/source/man.rst

```diff
@@ -559,6 +559,16 @@ The ``qpc report merge`` command combines results from two or more scan job iden
 
```

Comment:
In the above section called `Merging Scan Job Results` we need to add `--json-directory` to the command. ie:
```qpc report merge (--job-ids scan_job_identifiers | --report-ids report_identifiers | --json-files json_details_report_files | --json-directory path_to_directory_of_json_files)```

Also, change the `--json-directory=path_to_json_files` to `--json-directory=path_to_directory_of_json_files`